src/smolvm/vm.py

```
@@ -3283,9 +3292,7 @@ async def _close_log_when_done() -> None:
                 finally:
                     self._log_files.pop(key, None)
 
-        task = asyncio.create_task(_close_log_when_done())
-        self._background_tasks.add(task)
-        task.add_done_callback(self._background_tasks.discard)
+        asyncio.create_task(_close_log_when_done())
```

Ruff (0.15.15)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)